health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/ViewChecklist.js

```
@@ -0,0 +1,316 @@
+import MobileChecklist from "../../components/MobileChecklist";
+
+const ViewChecklist = () => {
+    const { t } = useTranslation();
+    const tenantId = Digit.ULBService.getCurrentTenantId();
+    const searchParams = new URLSearchParams(location.search);
+    const campaignName = searchParams.get("campaignName");
+    const role = searchParams.get("role");
+    const rlt = searchParams.get("role");
+    const roleLocal = (!rlt.startsWith("ACCESSCONTROL_ROLES_ROLES_")) ? "ACCESSCONTROL_ROLES_ROLES_" + rlt : rlt;
+    const checklistType = searchParams.get("checklistType");
+    let clt = searchParams.get("checklistType");
```

Biome

[error] This let declares a variable that is only assigned once.

'clt' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)